deploy/scripts/deploy-docker.sh

```diff
@@ -3,26 +3,28 @@
 DIR_PATH=$(pwd)
 CONF_FILES_DIR="conf-files"
 
-IMAGE_NAME="ufcglsd/iguassu"
+IMAGE_NAME="ufcglsd/iguassu:latest"
 CONTAINER_NAME="iguassu"
 
-MANAGER_PORT="8080"
-CONTAINER_PORT="9000"
+MANAGER_FRONT_PORT="9000"
```

Comment:
are these ports usually defined in bash or in a conf file?

Reply:
In the bash. Ports are defined by the application, so they are already used in bash.

Reply:
sounds odd. Anyway, I'll approve.

Reply:
It's a better practice remove the hardcoded configuration from scripts: more flexibility and customization without changing the source code. Move definition ports and the private key file name to a conf file.